microshift_install/microshift-embed-in-rpm-ostree.adoc

```diff
@@ -6,7 +6,18 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-You can embed {microshift-short} into a {op-system-ostree-first} {op-system-version} image. Use this guide to build a {op-system} image containing {microshift-short}.
+You can embed {microshift-short} into a {op-system-ostree-first} image. Use this guide to build a {op-system} image containing {microshift-short}.
+
+include::modules/microshift-install-system-requirements.adoc[leveloffset=+1]
+
+Plan to use a supported version of {op-system} paired with your version of {microshift-short} as described in the following table.
+
+[id="embed-ostree-rhde-compatibility-table"]
```

Comment:
@ShaunaDiaz Should this table include RedHat v 9.3 ?

Reply:
Yeah, those changes will land [here for 4.14](https://github.com/openshift/openshift-docs/pull/69956) and [here for 4.15](https://github.com/openshift/openshift-docs/pull/69960). This PR sets the stage for the branch updates.